src/PackageDependencyViewExtension/PackageDependencyView.xaml.cs

```diff
@@ -38,6 +41,18 @@ private Boolean HasMissingPackage
             }
         }
 
+        /// <summary>
+        /// Re-directs to a web link to get the feedback from the user. 
+        /// </summary>
+        private void ProvideFeedback(object sender, EventArgs e)
+        {
+            try {
+                System.Diagnostics.Process.Start(FeedbackLink);
+            }
+            catch (Exception ex) {
+                LogMessage.Info("Could not re-direct to the Dynamo forum page for feedback: "+ ex.Message);
```

Comment:
Can you add a simple dialog with the same language, also please add this to the resource dictionary and we should be able to localize it

Reply:
Added this.